packages/legend-lego/style/_data-grid.scss

```diff
@@ -16,3 +16,7 @@
 
 @import url('@ag-grid-community/styles/ag-grid.css');
 @import url('@ag-grid-community/styles/ag-theme-balham.css');
+
+.ag-cell {
```

Comment:
I'm worried this might affect other places that uses the data-grid (i.e. consumer dashboard/data-space), have you checked?